components/addTask.vue

```diff
@@ -47,15 +49,47 @@ import { defineComponent } from '@nuxtjs/composition-api'
 
 export default defineComponent({
   emits: ['newTask'],
+  data(){
+    return{
+      inputText:""
+    }
+  },
   methods: {
-    addTask() {
+    async addTask() {
       /**
        * @todo Complete this function.
        * @todo 1. Send the request to add the task to the backend server.
        * @todo 2. Add the task in the dom.
        * @hint use emit to make a event that parent can observe
        */
+      console.log("debyg");
+      console.log(this.inputText);
```

Comment:
Remove all console log statements before submitting PR.
Ditto below.

Reply:
Ok,Should i comment all console logs or erase it.

Sometimes I have to use it for debugging.

Reply:
It's better to remove all console log statements before submitting PRs. When you submit code to a larger repo which is open-source, you don't want these unnecessary statements to sneak into the codebase.